docs/tutorials/scheduling.rst

```diff
@@ -0,0 +1,269 @@
+.. _scheduling_tutorial:
+
+.. doctest-skip-all
+
+***************************
+Scheduling an Observing Run
+***************************
+
+.. note::
+
+    Some terms used have been defined in :ref:`Terminology <terminology>`.
+
+Contents
+========
+
+* :ref:`scheduling-defining_targets`
+
+* :ref:`scheduling-creating_constraints_and_observing_blocks`
+
+* :ref:`scheduling-creating_a_transitioner`
+
+* :ref:`scheduling-scheduling`
+
+.. _scheduling-defining_targets:
+
+Defining Targets
+================
+
+We want to observe Deneb and M13 in the B, V and R filters. We are scheduled
+for the first half-night on July 6 2016 and want to know the order we should
+schedule the targets in.
+
+First we define our `~astroplan.Observer` object (where we are observing from):
+
+.. code-block:: python
+
+    >>> from astroplan import Observer
+
+    >>> apo = Observer.at_site('apo')
+
+Now we want to define our list of targets (`~astroplan.FixedTarget` objects),
+any object that is in SIMBAD can be called by an identifier.
+
+.. code-block:: python
+
+    >>> from astroplan import FixedTarget
+
+    >>> deneb = FixedTarget.from_name('deneb')
+    >>> m13 = FixedTarget.from_name('m13')
+
+    >>> deneb
+    <FixedTarget "deneb" at SkyCoord (ICRS): (ra, dec) in deg (310.35797975, 45.28033881)>
+
+    >>> m13
+    <FixedTarget "m13" at SkyCoord (ICRS): (ra, dec) in deg (250.423475, 36.4613194)>
+
+We also need to define when we will be observing the targets, `~astropy.time.Time`
+objects for the start and end of our observing window. The half night goes from 7PM
+local time to 1AM local time, in UTC this will be from 2AM to 8AM.
+
+.. code-block:: python
+
+    >>> from astropy.time import Time
+
+    >>> start_time = Time('2016-06-07 02:00')
+    >>> end_time = Time('2016-06-07 08:00')
```

Comment:
Make these times the sunset time and the sunrise time, or noon before and noon after.